### PR TITLE
Python3 arch

### PR DESCRIPTION
**Pull request by looooo**

### CODE REVIEW

File: src/Mod/Arch/ArchProfile.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def readPresets():
     for profilefile in profilefiles:
         if os.path.exists(profilefile):
             try:
-                with open(profilefile, 'rb') as csvfile:
+                with open(profilefile) as csvfile:
```
Comment (wwmayer) [inline src/Mod/Arch/ArchProfile.py]: Are you sure to drop the rb flag?

Comment (looooo) [inline src/Mod/Arch/ArchProfile.py]: Hmm. I remember anything wasn't working with "rb". I think it was with the tests in python3. But this was some time ago ...
Maybe we should have  a eye on this.

Comment (looooo) [inline src/Mod/Arch/ArchProfile.py]: reverted to "rb"

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**